src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
Yes, it passes the manual test proposed in the commit message.

The div tag was added later as a bug fix for this commit even before it was pushed to GitHub.
Please take a look at this file from Rebuild: https://github.com/jenkinsci/rebuild-plugin/blob/master/src/main/resources/com/sonyericsson/rebuild/RebuildAction/StringParameterValue.jelly